pkg/apis/admissionregistration/v1alpha1/defaults.go

```diff
@@ -57,3 +58,19 @@ func SetDefaults_ParamRef(obj *admissionregistrationv1alpha1.ParamRef) {
 		obj.ParameterNotFoundAction = &v
 	}
 }
+
+// SetDefaults_MutatingAdmissionPolicySpec sets defaults for MutatingAdmissionPolicySpec
+func SetDefaults_MutatingAdmissionPolicySpec(obj *admissionregistrationv1alpha1.MutatingAdmissionPolicySpec) {
+	if obj.FailurePolicy == nil {
+		policy := admissionregistrationv1alpha1.Fail
+		obj.FailurePolicy = &policy
+	}
+}
+
+// SetDefaults_Mutation sets defaults for Mutation
+func SetDefaults_Mutation(obj *admissionregistrationv1alpha1.Mutation) {
+	if obj.ReinvocationPolicy == nil {
+		never := admissionregistrationv1.NeverReinvocationPolicy
+		obj.ReinvocationPolicy = &never
+	}
+}
```

Comment:
Looks like it was done this way because mutating webhooks defined reinvocation policy this way.

That said, reinvocationPolicy was added to webhooks after the API was already established.  I'm OK with changing it if there are no objections to having the field behave differently than how it does on mutating webhooks.

Reply:
Switched to a required field here.